vita3k/gui/src/themes.cpp

```diff
@@ -406,36 +406,36 @@ void init_themes(GuiState &gui, HostState &host) {
             pugi::xml_document doc;
 
             if (doc.load_file(theme_path_xml.c_str())) {
-                const auto infomation = doc.child("theme").child("InfomationProperty");
+                const auto information = doc.child("theme").child("InfomationProperty");
```

Comment:
like i said, revert this file, i want keep const value named like child ^^

Reply:
sony made mistakes, we aren't following them, right? right?

Reply:
i am no sur is error seriously haha and i just want keep same name for more easy follow